wiz/README.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Wiz Buildkite Plugin Template
+description: Utilize the Wiz Buildkite Plugin for scanning in your Buildkite pipelines.
```

Comment:
How about stealing the description from the plugin?


```suggestion
description: Scan your infrastructure-as-code CloudFormation stacks or Docker images for security vulnerabilities using Wiz.
```

Reply:
done